common/djangoapps/entitlements/rest_api/v1/serializers.py

```diff
@@ -50,7 +50,7 @@ class CourseEntitlementSupportDetailSerializer(serializers.ModelSerializer):
         slug_field='username',
         default=serializers.CurrentUserDefault()
     )
-    unenrolled_run = CourseKeyField('unenrolled_run.id')
+    unenrolled_run = CourseKeyField(source='unenrolled_run.id')
```

Comment:
unrelated to the goal of the pr, but this line was breaking my migrations locally.

Reply:
should this get `@medality_custom` added to it for good measure?

Reply:
good call. added.